tests/foreman/api/test_host.py

```diff
@@ -1534,50 +1534,3 @@ def test_positive_bulk_destroy(self, module_org):
         for host_id in host_ids[:-1]:
             with pytest.raises(HTTPError):
                 entities.Host(id=host_id).read()
-
-
-class TestHostTraces:
-    """Tests for host tracer"""
-
-    @pytest.mark.tier4
-    @pytest.mark.rhel_ver_match('[^6].*')
-    def test_positive_tracer_list_and_resolve(self, tracer_host):
-        """Install tracer on client, downgrade the service, check from the satellite
-        that tracer shows and resolves the problem. The test works with a package specified
-        in settings. This package is expected to install a systemd service which is expected
-        to log its version to /var/log/{package}/service.log.
-        The rpm is not supposed to start the service upon install.
-
-        :id: 552fc212-536a-11ec-865d-98fa9b6ecd5a
-
-        :expectedresults: Tracer detected and resolved the problem,
-            the affected service was restarted
-
-        :parametrized: yes
-
-        :CaseImportance: Medium
-        """
-        host = tracer_host.nailgun_host
-        package = settings.repos["MOCK_SERVICE_RPM"]
-        # mark the service log messages for later comparison and downgrade the pkg version
-        service_ver_log_old = tracer_host.execute(f'cat /var/log/{package}/service.log')
-        package_downgrade = tracer_host.execute(f'yum -y downgrade {package}')
-        assert package_downgrade.status == 0
-
-        # tracer should detect a new trace
-        traces = host.traces(data={'host_id': host.id})
```

Comment:
question, though API and CLI tests look similar, here we execute an endpoint to find new traces and resolve traces, but in CLI we've commands to the same, it doesn't look redundant, so why it should be removed?

Reply:
The idea here is that since hammer visits those api endpoints anyway, it would be hit as well if anything happened to these endpoints. So the overlap between api and cli is significant